### PR TITLE
Fix wrong use of class constants

### PR DESCRIPTION
**Pull request by Alkarex**
http://php.net/manual/language.oop5.constants.php

### CODE REVIEW

**Comment by Alkarex** [comment]
I had a bunch of errors of the following type (PHP 7.2.3 / Ubuntu 18.04 LTS)

```
[Sun May 13 04:18:46 2018] [error] [pid 13481] mod_proxy_fcgi.c(848): [client 163.172.11.188:34408] AH01071: Got error 'PHP message: PHP Warning:  Use of undefined constant DFRN_REPLY - assumed 'DFRN_REPLY' (this will throw an Error in a future version of PHP) in /.../src/Protocol/DFRN.php
 on line 2216\nPHP message: PHP Warning:  Use of undefined constant DFRN_REPLY - assumed 'DFRN_REPLY' (this will throw an Error in a future version of PHP) in /.../src/Protocol/DFRN.php on line 2637
```

**Comment by tobiasd** [comment]
Could you change the target of the PR to the 2018.05-rc branch? this is clearly a bugfix and not a new feature.

**Comment by Alkarex** [comment]
Here you are; @tobiasd

File: src/Protocol/DFRN.php [hunk 1]
```diff
@@ -2141,7 +2141,7 @@ private static function updateContent($current, $item, $importer, $entrytype)
 
 			$changed = true;
 
-			if ($entrytype == DFRN_REPLY_RC) {
+			if ($entrytype == DFRN::DFRN_REPLY_RC) {
```
Comment (MrPetovan) [inline src/Protocol/DFRN.php]: It should be `DFRN::REPLY_RC` instead, can you please rename all `DFRN_*` constants to just `*` and modify their use?

Comment (Alkarex) [reply src/Protocol/DFRN.php]: Done https://github.com/friendica/friendica/pull/5036/commits/736c76d068742a4e93bef77d1f23a0c9e23417be

**Comment by MrPetovan** [comment]
Pleasure doing business with you sir!

**Pull request merged**